libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-party-b2b-access.store.ts

```diff
@@ -19,15 +19,20 @@ import { ComponentStore } from '@ngrx/component-store';
 import { tapResponse } from '@ngrx/operators';
 import { Observable, switchMap, exhaustMap, tap, finalize, withLatestFrom, of } from 'rxjs';
 
-import {
-  MarketParticipantActorCredentialsDto,
-  MarketParticipantActorHttp,
-} from '@energinet-datahub/dh/shared/domain';
+import { MarketParticipantActorHttp } from '@energinet-datahub/dh/shared/domain';
+
+import { lazyQuery } from '@energinet-datahub/dh/shared/util-apollo';
+
 import { HttpErrorResponse } from '@angular/common/http';
 import { ApiErrorCollection } from '@energinet-datahub/dh/market-participant/data-access-api';
+import { GetActorCredentialsDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import type { ResultOf } from '@graphql-typed-document-node/core';
+
+type ActorCredentials = ResultOf<typeof GetActorCredentialsDocument>['actorById'];
 
 interface DhB2BAccessState {
-  credentials: MarketParticipantActorCredentialsDto | null;
+  credentials: ActorCredentials | null | undefined;
```

Comment:
suggestion: if we use `ActorCredentialsDto` type here, there will be no need to use `credentials?.credentials` (double `credentials`) later on.

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3376/commits/54b8cfe8147febd03616088ca42d951a8486c176